### ISSUE TITLE
Add array jobs section to Scheduler section.

### ISSUE THREAD

**Issue by hgputnam** [UCSF-CBI/c4#61]
I got some questions from a user wanting to learn how to submit array jobs. I have never done that and don't really know how, I realized that our docs site doesn't have any info for that. @HenrikBengtsson how fluent are you with array jobs? I will teach myself how to use them and write up a section but it would be nice if someone who knows what they are doing could check my work.

**Comment by HenrikBengtsson** [comment]
I agree, we should have a section, or possibly a page, on how to do array jobs in Slurm. Yes, I'm happy to review. I don't have anything ready for this, and honestly, I haven't had to use them for Slurm.  The sale pitch is that it allows the user to re-use the same job script for lots of jobs, instead of copy-edit-save the same job over and over again.  FWIW, the <https://ucsf-c4.slack.com/archives/C01M3CSSZT3/p1642031088014900?thread_ts=1642014448.014800&cid=C01M3CSSZT3> answer illustrates how to use them.
